vectordb_bench/backend/task_runner.py

```diff
@@ -161,7 +161,7 @@ def _load_train_data(self):
         finally:
             runner = None
 
-    def _serial_search(self) -> tuple[float, float]:
+    def _serial_search(self) -> tuple[float, float, float]:
         """Performance serial tests, search the entire test data once,
         calculate the recall, serial_latency_p99
```

Comment:
Maybe also change the docstring? TBH I think the avg latency should be in a seperate PR ;)